src/annotator/test/integration/hypothesis-injector-test.js

```diff
@@ -24,43 +23,39 @@ describe('CrossFrame multi-frame scenario', () => {
       call: sandbox.stub(),
       destroy: sandbox.stub(),
     };
-    fakeAnnotationSync = { destroy: sandbox.stub() };
-    proxyAnnotationSync = sandbox.stub().returns(fakeAnnotationSync);
-    proxyBridge = sandbox.stub().returns(fakeBridge);
-
-    $imports.$mock({
-      './annotation-sync': { AnnotationSync: proxyAnnotationSync },
-      '../shared/bridge': proxyBridge,
-    });
+    hypothesisInjectors = [];
 
     container = document.createElement('div');
     document.body.appendChild(container);
-
-    config = {
-      clientUrl: 'data:,', // empty data uri
-    };
-
-    crossFrame = null;
   });
 
   afterEach(() => {
     sandbox.restore();
-    crossFrame.destroy();
-    container.parentNode.removeChild(container);
-
-    $imports.$restore();
+    hypothesisInjectors.forEach(hypothesisInjector =>
+      hypothesisInjector.destroy()
+    );
```

Comment:
This is OK, although when the meaning of a variable in a small closure is very obvious (eg. in an array `map` or `forEach` callback) I tend to keep it short. eg. `hypothesisInjectors.forEach(injector => injector.destroy())`